packages/tools/generator-component/generators/app/templates/__package__.json

```diff
@@ -38,8 +38,9 @@
     "lint:style": "vue-cli-service lint:style"<% } else { %>,
     "test:coverage": "jest test:unit --coverage"<% } %>,
     "test": "vue-cli-service test:unit"<% if(config.needsComponentTests) { %>,
-    "test-component:chrome": "cross-env-shell TEST_TYPE=component wdio ../../../../wdio-chrome.conf.js",
-    "test-a11y:chrome": "cross-env-shell TEST_TYPE=a11y wdio ../../../../wdio-chrome.conf.js"<% } %>
+    "test-component:chrome": "cross-env-shell TEST_TYPE=component wdio ../../../../wdio-chrome.conf.js"<% } %>,
+    "test-a11y:chrome": "cross-env-shell TEST_TYPE=a11y wdio ../../../../wdio-chrome.conf.js"<% if(config.needsVisualTests) { %>,
```

Comment:
I'm guessing this will include this line even if the component is a service potentially? I think it probably needs to be wrapped in the above `needsComponentTests` if statement?

Reply:
I'm not sure - does this line solve that? https://github.com/justeat/fozzie-components/pull/1525/files#diff-22e5b67bfabcb4c42acfca9714ab1d95874f60b9bc33a20726363824807fb021R41 
I was wondering about putting it inside `needsComponentTests` but kept them separate in case there's a scenario where we'll need visual tests but not component tests. Happy to change though

Reply:
I think the `test-a11y:chrome` script isn't wrapped in any if statement atm though, so it'll always be included in the package.json atm?

Reply:
Yeh, that'd be good. Just need to make sure it doesn't output for services, so as long as that case is covered then I'll be happy 👍

Reply:
awesome, thank you - just fixed 👍